R/codemeta_description.R

```diff
@@ -1,12 +1,20 @@
-options(codemeta_context = "https://doi.org/10.5063/schema/codemeta-2.0")
+# Define gloabl string constant
```

Comment:
Small typo, global.
I'd make it a function and lower-case
```r
url_codemeta_schema <- function(){
  "https://doi.org/10.5063/schema/codemeta-2.0"
  }
```

Reply:
Good idea. I will do that. What do you mean with "typo"? Wrong spelling somewhere?
Is the option actually required or can we then just call the new function instead?

Reply:
yes a typo in the comment above the function "Define gloabl string"

I am not sure about the option, I think @cboettig added it so that the user can toggle it globally.